src/lambda_function.py

```diff
@@ -54,3 +56,39 @@ def check_prime(num):
             return False
         sq_root += 1
     return True
+
+
+def sync_ddb_table(source_ddb, destination_ddb):
+    response = source_ddb.scan(
+        TableName="table1"
+        )
+    for item in response['Items']:
+        destination_ddb.put_item(
+            TableName="table2", Item=item
+            )
+
+
+def list_sns(region, creds, sns_topics=[]):
+    sns = boto_session('sns', creds, region)
+    response = sns.list_topics()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `list_topics` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `IsTruncated`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)